api/all/src/main/java/io/opentelemetry/api/OpenTelemetry.java

```diff
@@ -21,6 +21,20 @@
  * @see ContextPropagators
  */
 public interface OpenTelemetry {
+  /** Returns a default, completely no-op {@link OpenTelemetry}. */
+  static OpenTelemetry getDefault() {
+    return DefaultOpenTelemetry.noop();
+  }
+
+  /**
+   * Returns an {@link OpenTelemetry} which will do remote propagation of {@link
+   * io.opentelemetry.context.Context} using the provided {@link ContextPropagators} and is no-op
+   * otherwise.
+   */
+  static OpenTelemetry getPropagating(ContextPropagators propagators) {
```

Comment:
`getPropagationOnly`?

Reply:
It's a bit obtuse - propagating looks great next to noop I think since it's just the version that adds propagation so think the only is naturally implied. Compared to default, though, maybe not as much

Reply:
There is one corner case where it's a bit misleading to say "only"...if the auto-instrumentation agent instruments the API, you'll get full functionality here. Actually...in that case will the propagators provided here be ignored and replaced with the agent's internal configuration?

Reply:
Are you? Why not via the Global?

Reply:
If you use the Global, the agent also instruments it and uses it's own internal SDK instance.

Reply:
@jkwatson Yeah you mentioned the agent overriding the API trick - do you know what version it was? Will confirm, but it should have changed to only override what's returned by `GlobalOpenTelemetry.get()`, will confirm. That's the behavior we want anyways

Reply:
I don't remember what version it was. probably 13 or 14